cypress.config.ts

```diff
@@ -8,6 +8,9 @@ export default defineConfig({
       codeCoverageTask(on, config)
       return config
     },
+    supportFile: 'cypress/support/e2e.ts',
```

Comment:
[SUG] Maybe we should add `testIsolation: false` to the config so we can use `before` instead of `beforeEach` (in some cases we still will have to use `beforeEach`) in the e2e tests and avoid doing rerenders in each test iteration. Although this could break some tests.
```suggestion
    supportFile: 'cypress/support/e2e.ts',
    testIsolation: false,
```

Reply:
definitely! Actually, I didn't know how to do this, that's why I used beforeEach instead of before in all my tests jaja, thank you!